config/owasp/suppressions.xml

```diff
@@ -27,6 +27,10 @@
   <suppress>
     <notes>spring-*-5.3.12</notes>
     <cve>CVE-2021-22060</cve>
+    <cve>CVE-2022-22950</cve>
+    <cve>CVE-2013-4152</cve>
+    <cve>CVE-2014-0054</cve>
```

Comment:
Can you please create a Jira for fixing these CVEs?

Reply:
https://tools.hmcts.net/jira/browse/ISDB-141 has been created for same